service/src/main/java/org/broadinstitute/listener/relay/Utils.java

```diff
@@ -18,6 +20,15 @@ public class Utils {
   public static final String TOKEN_NAME = "LeoToken";
   public static final String SET_COOKIE_API_PATH = "setcookie";
 
+  /**
+   * Paths which should not trigger dateAccessed runtime updates. TODO: move related logic into
+   * SetDateAccessed* classes.
```

Comment:
I would love to do this (encapsulate logic in the SetDateAccessedListener instead of leaving it hanging out in Utils).

I would also like to make the paths configurable and have Leo pass them in.

I don't think these beautifications should block this critical bugfix, but maybe they can be addressed when we add a new non-keep-alive path to the runtime metrics monitor.